server/middlewares/middleware.spec.js

```diff
@@ -0,0 +1,21 @@
+import sinon from 'sinon';
+import model from '<models>';
+import { verifyUser } from '<middlewares>';
+import { inputObject } from '<fixtures>/data';
+
+const { User } = model;
+
+describe('middlewares', () => {
+  it('should return server error in catch block', async () => {
+    const req = { body: { password: 123456 }, sanitizedData: inputObject };
+    const res = {
```

Comment:
Remember the thing we did up there ☝️with `res`, this is one of them 😉 
Also, ensure that you update/refactor its reference to Sinon

```
    sinon.stub(res, 'status').returnsThis();
    sinon.stub(User, 'create').throws();
    await verifyUser(req, res);
    expect(res).toBeTruthy();
```